src/apis.php

```diff
@@ -1750,8 +1750,11 @@ protected function getCurlOptArray($override = array())
             )
                 ? SHOPGATE_PLUGIN_VERSION
                 : 'called outside plugin');
-        $opt[CURLOPT_SSL_VERIFYPEER] = false;
         $opt[CURLOPT_RETURNTRANSFER] = true;
+		$opt[CURLOPT_SSL_VERIFYPEER] = true; // *always* verify peers, otherwise MITM attacks are trivial
+		// Use value of CURL_SSLVERSION_TLSv1_2 for CURLOPT_SSLVERSION, because it is not available before PHP 5.5.19 / 5.6.3
+		// Actual usage of TLS 1.2 (which is required by PCI DSS) depends on PHP cURL extension and underlying SSL lib
+		$opt[CURLOPT_SSLVERSION]     = 6;
```

Comment:
What happens when the merchant has a PHP Version <= 5.5.18 or <= 5.6.2 installed on his server?

Reply:
Discussed that with Patrick, so I can answer:
With the constant it would break, with the number it depends in the underlying cURL & SSL library what happens. It could break if that version wasn't supporting TLS 1.2. On the other hand the Shopgate Merchant API is rejecting connections with TLS < 1.2 anyway. So it would break either way.

Reply:
ok

Reply:
That's why we are using the int value instead of the constant. As long as curl supports the option, everything is fine - it forces curl to use TLS 1.2 even on systems that would otherwise try to connect with TLS 1.0 (older defaults). If curl does not support TLS 1.2, the merchant can't connect to us anyways.

Reply:
My question was not about using the constant CURLOPT_SSLVERSION. It was about what will happen on older systems when you pass the int 6

Reply:
Ninja'd... The same change already solved ticket LIBRARY-170 (shop using PHP 5.4.16 with NSS 3.28.4), so it should be ok.